### DISCUSSION TITLE
DGS Guide Chinese

### DISCUSSION THREAD

**Discussion by ferrisyang** [Netflix/dgs-framework#335] [Show and tell]
Hi，I translate DGS guide into Chinese, and public the doc by gitbook (https://ferris-yang.gitbook.io/netflix-dgs-chinese/v/main/).
So, I have 2 requests：
1. Could you give me the DGS guide doc Github url？I can watch it, and update my doc in a timely manner.
2. If possible, my Chinese doc for DGS can be a part of Official Guide, what do you think?

**Comment by paulbakker** [comment]
That's so cool! Thanks for putting in that effort.

The git repo for the docs is here: https://github.com/netflix/dgs
